shared/getting-started/getDeviceOnDash/coral-dev.md

```diff
@@ -0,0 +1,7 @@
+Unplug the **{{ $device.name }}** and [change the boot mode switches][boot-switches] to boot from the SD card. Insert the SD card into your **{{ $device.name }}** and then power on the board using a 2-3A power cable connected to the USB-C port labeled "PWR". Warning, this will also completely erase internal storage media, so please make a backup first.
```

Comment:
```suggestion
Unplug the **{{ $device.name }}** and [change the boot mode switches][boot-switches] to boot from the SD card. Insert the SD card into your **{{ $device.name }}** and then power on the board using a 2-3A power cable connected to the USB-C port labeled "PWR". 

__NOTE__: Warning, this will also completely erase internal storage media, so please make a backup first.
```

Reply:
I'll update this to use `__Warning:__` instead